src/appsignal/config.py

```diff
@@ -104,6 +108,38 @@ class Options(TypedDict, total=False):
 )
 
 
+def validate(config: Options):
+    config["log_file_path"] = log_file_path(config)
+
+
+def log_file_path(config: Options) -> Optional[str]:
+    filename = "appsignal.log"
+    path = config.get("log_path")
+
+    if path and not os.access(path, os.W_OK):
+        path = None
+        print(
+            (
+                f"appsignal: Unable to write to configured '{path}'. Please check the "
+                "permissions of the 'log_path' directory."
+            )
+        )
+
+    if not path:
+        path = "/tmp" if platform.system() == "Darwin" else tempfile.gettempdir()
+
+    if not os.access(path, os.W_OK):
+        print(
+            (
+                f"appsignal: Unable to write to '{path}'. Please check the "
+                "permissions of the 'log_path' directory."
+            )
+        )
+        return None
```

Comment:
If I recall correctly, other integrations set the `log` config options (which determines where to write logs) to `stdout` instead of `file` when the logging path is not writable. I see that `log` is not implemented yet, should this be done as part of #34?